tools/rodent_all.csv

```diff
@@ -0,0 +1,237 @@
+"period","BA","DM","DO","DS","NA","OL","OT","PB","PE","PF","PH","PI","PL","PM","PP","RF","RM","RO","SF","SH","SO","total","newmoonnumber","year","month"
```

Comment:
I'm not sure that we need to commit these files (all three that are currently being written to `tools`) if they're being auto-generated on each run.

Reply:
We've decided to move the 3 data files to a data directory and have them committed in the weekly runs